### PR TITLE
Updating the HDMI output note to reflect output starting v2.0 on Raspi devices

### PR DESCRIPTION
**Pull request by nitishagar**
Issue: https://github.com/balena-io/docs/issues/478

### CODE REVIEW

File: shared/getting-started/getDeviceOnDash/raspberry-pi.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ If your device still hasn't shown up on your dashboard after 10 minutes, somethi
 
 If you still can't get your device online, come on over and talk to us on our [support channel][usingSupport].
 
-__Note:__ If you have an HDMI screen attached, you should see `"Booted - Check your {{ $names.company.lower }} dashboard."` on the screen when the device boots. If instead you see rainbow colors or a black screen with a raspberry on it, it could mean that the SD card was not burned correctly or is corrupted.
+__Note:__ If you have an HDMI screen attached, you should see Balena logo on the screen when the device boots. on the screen when the device boots. If instead you see rainbow colors or a black screen with a raspberry on it, it could mean that the SD card was not burned correctly or is corrupted.
```
Comment (lurch) [inline shared/getting-started/getDeviceOnDash/raspberry-pi.md:18]: "on the screen when the device boots. on the screen when the device boots."

Whoops... :wink:

Comment (nitishagar) [reply shared/getting-started/getDeviceOnDash/raspberry-pi.md]: Good catch.. Can never trust a good "replace all"!

**Review by vipulgupta2048 — COMMENTED** [review]
Please refer to the style guide while making change to the documentation. Balena almost everywhere should be referred to as `balena`. https://github.com/balena-io/docs#balena-specific-words

**Comment by nitishagar** [comment]
Thanks @vipulgupta2048. Updated based on naming convention (corrected some other references in process). :)

**Review by vipulgupta2048 — COMMENTED** [review]
LGTM, merge this PR by issuing the `I self-certify!` command to approve the PR. 
Reference: https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#balenistas-only-using-versionbot-to-manage-your-pull-request

**Comment by nitishagar** [comment]
@balena-ci I self-certify!

**Pull request merged**